src/ColumnNameSanitizer.php

```diff
@@ -15,7 +15,7 @@ class ColumnNameSanitizer
      * Column names are alphanumeric strings that can contain
      * underscores (`_`) but can't start with a number.
      */
-    private const VALID_COLUMN_NAME_REGEX = '/^(?![0-9])[A-Za-z0-9_-]*$/';
+    private const VALID_COLUMN_NAME_REGEX = '/^(?![0-9])[A-Za-z0-9._-]*$/';
```

Comment:
Does this dot need to be escaped in order to represent the string literal `.`?

Reply:
It works as is

Reply:
it should be escaped sense currently it would allow any character

Reply:
I'm happy to add the escape, but as far as I was aware this is fine. Have also tested it in practical use and it does work as intended and continues to block other characters.

Without escape
---
<img width="400" alt="Screenshot 2019-04-11 at 04 17 25" src="https://user-images.githubusercontent.com/4124930/55918903-7c3bbd00-5c11-11e9-9564-a29914b5787c.png">

With escape
---
<img width="383" alt="Screenshot 2019-04-11 at 04 17 33" src="https://user-images.githubusercontent.com/4124930/55918904-7c3bbd00-5c11-11e9-9435-f295d0a7a2bc.png">

Reply:
Inside a regex class (the square brackets), dots and most otherwise special characters lose their special meanings.

`.` Matches any char  
`[.]` only matches the dot  

You can check this [here](https://regex101.com/r/y0hl1u/1)